readme/Index.scalatex

```diff
@@ -56,22 +56,27 @@
     If you want some initialization code available to the REPL, you can add it to your @code{~/.ammonite/predef.scala}.
 
   @p
-    You can also try out Ammonite in an existing SBT project, add the following to your @code{build.sbt}
+    You can also try out Ammonite in an existing SBT project, add the following to @code{project/plugins.sbt}
 
   @hl.scala
-    libraryDependencies += "com.lihaoyi" % "ammonite-repl" % "@ammonite.Constants.version" % "test" cross CrossVersion.full
-
-    initialCommands in (Test, console) := """ammonite.repl.Repl.run("")"""
+    addSbtPlugin("com.lihaoyi" % "ammonite-sbt" % "@ammonite.Constants.version")
 
   @p
     After that, simple hit
   @code
-    sbt projectName/test:console
+    sbt projectName/ammonite:run
   @p
     To activate the Ammonite REPL
 
   @p
-    You can also pass a string to the @code{run} call containing any commands or imports you want executed at the start of every run. If you want Ammonite to be available in all projects, simply add the above snippet to a new file @code{~/.sbt/0.13/global.sbt}.
+    Hit
+  @code
+    sbt projectName/ammonite-test:run
```

Comment:
Is it all possible to make `ammonite:run` and `ammonite:test` do the right thing? If not that's ok, but that certainly sounds like a better place to be in if we can make it happen